spec/lcm/integration/support/project_helper.rb

```diff
@@ -87,6 +87,35 @@ def deploy_processes(ads)
           de_synchronize_all: true,
           state: 'DISABLED'
         )
+
+        component_data = {
+          name: 'lcm-end-to-end-sql-exec',
+          type: :etl,
+          component: {
+            name: 'gdc-etl-sql-executor',
+            version: '1'
+          }
+        }
+        project.deploy_process component_data
```

Comment:
info: Refers to 'project' more than self (maybe move it to another class?) (FeatureEnvy) (reek)